src/Definition/Entity.php

```diff
@@ -137,6 +149,9 @@ public function setRepository(?string $repository): self
         return $this;
     }
 
+    /**
+     * @return class-string|null
```

Comment:
Shouldn't it be some `class-string<RepositoryInterface>|null`?

Reply:
Added ```class-string<RepositoryInterface>|null```